ui/app/models/secret-engine.js

```diff
@@ -60,30 +67,28 @@ export default SecretEngineModel.extend({
     helperTextEnabled: 'Delete all new versions of this secret after',
   }),
 
-  modelTypeForKV: computed('engineType', 'options.version', function () {
+  modelTypeForKV: computed('engineType', 'version', function () {
     let type = this.engineType;
-    let version = this.options?.version;
     let modelType = 'secret';
-    if ((type === 'kv' || type === 'generic') && version === 2) {
+    if ((type === 'kv' || type === 'generic') && this.version === 2) {
       modelType = 'secret-v2';
     }
     return modelType;
   }),
 
   isV2KV: equal('modelTypeForKV', 'secret-v2'),
 
-  formFields: computed('engineType', 'options.version', function () {
+  formFields: computed('engineType', 'version', function () {
     let type = this.engineType;
-    let version = this.options?.version;
     let fields = ['type', 'path', 'description', 'accessor', 'local', 'sealWrap'];
     // no ttl options for keymgmt
     const ttl = type !== 'keymgmt' ? 'defaultLeaseTtl,maxLeaseTtl,' : '';
     fields.push(`config.{${ttl}auditNonHmacRequestKeys,auditNonHmacResponseKeys,passthroughRequestHeaders}`);
```

Comment:
as a `belongsTo` attr will this syntax for `config` still works as expected right? Classic models look so funky to me with all the curly-bracket-strings hanging around, so just wanted to check!

Reply:
Yes this a pattern used by the `field-to-attrs` util where it is passing the strings into [expandProperties](https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/expandProperties) which handles the brace expansion. Basically instead of having to write `config.auditNonHmacRequestKeys, config.passthroughRequestHeaders` you can define it using brace expansion and `expandProperties` will take care of it for you. Check out the link for examples.

Reply:
Thank you!